doc/rbd/iscsi-initiator-linux.rst

```diff
@@ -74,40 +74,46 @@ Install the iSCSI initiator and multipath tools:
 
 #. Discover the target portals:
 
+   .. prompt:: bash #
+
+      iscsiadm -m discovery -t st -p 192.168.56.101
+
    ::
 
-       # iscsiadm -m discovery -t st -p 192.168.56.101
        192.168.56.101:3260,1 iqn.2003-01.org.linux-iscsi.rheln1
        192.168.56.102:3260,2 iqn.2003-01.org.linux-iscsi.rheln1
 
-#. Login to target:
+#. Login to the target:
 
-   ::
+   .. prompt:: bash #
 
-       # iscsiadm -m node -T iqn.2003-01.org.linux-iscsi.rheln1 -l
+      iscsiadm -m node -T iqn.2003-01.org.linux-iscsi.rheln1 -l
 
 **Multipath IO Setup:**
 
-The multipath daemon (``multipathd``), will set up devices automatically
-based on the ``multipath.conf`` settings. Running the ``multipath``
-command show devices setup in a failover configuration with a priority
-group for each path.
+#. The multipath daemon (``multipathd``) uses the ``multipath.conf`` settings
+   to set up devices automatically. Running the ``multipath`` command shows the
```

Comment:
@idryomov: Could you give me more detail on what "Running the ``multipath`` command show devices setup in a failover configuration with a priority group for each path" is trying to express? 

It'd be helpful to have an explanation of what "priority group" means.

Reply:
I think it's just trying to say that each path is placed into its own priority group.  Here is a good overview: https://documentation.suse.com/sles/15-SP1/html/SLES-all/cha-multipath.html#sec-multipath-grouping